src/inference/INCLUDE/evaluate.py

```diff
@@ -126,44 +122,60 @@ def inference(dataloader, model, device, label_map):
 
     return predictions
 
-
-video_paths = glob.glob(os.path.join(args.data_dir, "*"))
-save_dir = "keypoints_dir"
-if os.path.isdir(save_dir):
-    shutil.rmtree(save_dir)
-os.mkdir(save_dir)
-for path in tqdm(video_paths, desc="Processing Videos"):
-    process_video(path, save_dir)
-
-label_map = load_label_map("include")
-dataset = KeypointsDataset(
-    keypoints_dir=save_dir,
-    max_frame_len=169,
-)
-
-dataloader = data.DataLoader(
-    dataset,
-    batch_size=1,
-    shuffle=False,
-    num_workers=4,
-    pin_memory=True,
-)
-label_map = dict(zip(label_map.values(), label_map.keys()))
-
-device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
-config = TransformerConfig(size="large", max_position_embeddings=256)
-model = Transformer(config=config, n_classes=263)
-model = model.to(device)
-
-pretrained_model_name = "include_no_cnn_transformer_large.pth"
-pretrained_model_links = load_json("pretrained_links.json")
-if not os.path.isfile(pretrained_model_name):
-    link = pretrained_model_links[pretrained_model_name]
-    torch.hub.download_url_to_file(link, pretrained_model_name, progress=True)
-
-ckpt = torch.load(pretrained_model_name)
-model.load_state_dict(ckpt["model"])
-print("### Model loaded ###")
-
-preds = inference(dataloader, model, device, label_map)
-print(json.dumps(preds, indent=2))
+def get_inference_args(video_paths):
```

Comment:
Does this accept multiple video paths or just one path with single/multiple videos?

Reply:
This takes multiple paths, because the original design in INCLUDE passes multiple videos.
so in my api I call this by creating a list with only one video path